server/database/schema.sql

```diff
@@ -14,18 +14,17 @@ create table art (
   id int unsigned primary key auto_increment not null,
   title varchar(80),
   information text,
-  latitude DECIMAL (8,6) not null,
-  longitude DECIMAL (9,6) not null,
-  upload_date date not null,
-  status varchar(20) not null,
-  is_best_picture boolean
+  latitude DECIMAL(8,6) not null,
+  longitude DECIMAL(9,6) not null,
+  upload_date date not null DEFAULT (CURRENT_DATE),
+  status varchar(20) not null DEFAULT "pending"
 );
 
 create table picture (
   id int unsigned primary key auto_increment not null,
   image varchar(2048) not null,
   user_id int unsigned not null,
-  art_id int unsigned not null,
+  art_id int unsigned DEFAULT NULL,
```

Comment:
why null?
we said everytime a picture is added it automatically creates an art as well

Reply:
It's changed 👍